logic/common/tiled_utils.py

```diff
@@ -164,6 +164,22 @@ def RotateTileId(tile_id):
 #--------------------------------------------------#
 '''Object Properties'''
 
+def GetProperty( curr_obj, property_name ):
+    '''Extract the property as string, e.g. returns 'BRONZE' from GetProperty('_type')'''
+    for curr_property in curr_obj.find('properties').findall('property'):
+        if curr_property.get('name') == property_name:
+            return curr_property.get('value')
+    return ''    # returning None would crash when attempted to be converted into string
+
+
+def GetSort( curr_obj ):
+    '''Take an object and extract its sort value as string, e.g. "fg_tiles,-1"'''
+    return GetProperty(curr_obj, '_sort')
```

Comment:
I personally prefer being lazy whenever possible, since future me may not remember how the inner workings well enough to put all the arguments correctly blind.

The 1st benefit is that the input is simple and future-proof - You have 1 input argument, and it returns 1 string afterwards. Without it, we would have to remember (1) the function name, & (2) the in-editor keyword for searching. I almost always avoid using literal constant for string (i.e. '_sort'), unless it exists only within the current file, or is used in a common module. While not likely, if we are to change the property name from '_sort' to something else like _sort_value, we would have to check every single code file to change it (instead of only changing 1 string in this GetSort() function).

The 2nd benefit is that it's what I call a "shortcut function". It serves one simple purpose, is easily understood since it's mostly just the same function call, and is easy to used like I mentioned in the previous paragraph. Besides being convenient to use, it doesn't stem too far from the GetProperty() function and thus I don't think it would hurt the overall structure by causing confusion either.

The 3rd benefit is that its name is short and intuitive (thus easily memorised)... That's it. Again, usually I just want to be lazy and type as little as I can if possible.

... So yeah, future-proof & convenient. But if you're really against it...

Reply:
I could understand your reasoning more if this was a function that was used a lot - but it's used once.

How about for now, GetSort can exist, but it resides inside sort_conflict.py until such a time comes where its existence is proven to be useful in other tools as well?

Reply:
For personal projects, I would probably keep it there, just so future me would know to put more lazy shortcuts there whenever I feel like it, e.g. `GetName` and `GetType`.

But since it's your project, I'm just gonna remove it entirely. And like you said, it's only used once anyway.

Reply:
Okay I forgot what the function for getting the name of an object should be, so I now insist on adding these back in for future-me.